schema/config-linux.json

```diff
@@ -170,11 +170,14 @@
                             },
                             "period": {
                                 "id": "https://opencontainers.org/schema/bundle/linux/resources/cpu/period",
-                                "$ref": "defs.json#/definitions/uint64Pointer"
```

Comment:
These absolutely need to be `uint64Pointer`s. The reason for this is explained (in great detail) in #233.